### PR TITLE
Issue #SH-681 fix: Batch Details is displayed in the Pop up Post enrolment End date

### PR DESCRIPTION
**Pull request by souravdey091**

### CODE REVIEW

File: src/app/client/src/app/modules/learn/components/batch/batch-details/batch-details.component.html [hunk 1]
```diff
@@ -10,17 +10,17 @@
                 <div>{{resourceService?.frmelmnts?.lbl?.availableBatches}}</div>
                 <div>{{resourceService?.frmelmnts?.lbl?.courseCreatedBy}}</div>
             </div>
-            <div class="join-batch__body" *ngFor="let batch of allBatchList">
+            <span  *ngFor="let batch of allBatchList" >
+                <div class="join-batch__body" *ngIf="!isEnrollmentAllowed(batch?.enrollmentEndDate)">
```
Comment (harishgilimi) [inline src/app/client/src/app/modules/learn/components/batch/batch-details/batch-details.component.html:14]: Remove this in ts file itself

Comment (souravdey091) [reply src/app/client/src/app/modules/learn/components/batch/batch-details/batch-details.component.html:14]: Can't remove it as batch creator needs to see the batch whose enrollment date is over

Comment (harishgilimi) [reply src/app/client/src/app/modules/learn/components/batch/batch-details/batch-details.component.html:14]: you should differentiate assign it to another property with map function and use that here

Comment (souravdey091) [reply src/app/client/src/app/modules/learn/components/batch/batch-details/batch-details.component.html]: Done

**Pull request merged**